src/octopeer-github/test/ChromeMock.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Created by Mitchell on 27-5-2016.
+ * Mocks chrome modules, functions and properties.
+ * Because of this mock, test cases can be performed on chrome dependent items (e.g. the storage).
+ */
+
+let _window: any = window;
+
+/* tslint:disable:no-empty */
```

Comment:
Also Mitchell's branch. But all functions are empty, so it needs to be over the entire file.